config_models/admin.py

```diff
@@ -128,7 +128,7 @@ def has_output(self):
         """ Should this filter be shown? """
         return True
 
-    def choices(self, cl):
+    def choices(self, cl):  # pylint: disable=arguments-differ
```

Comment:
@snowcrshd You're so thoroughly ignoring my suggestions to make the interface consistent instead of suppressing an error :) You could at least say "no, to me unused argument is worse than mismatching signature, so I'll keep it" and I wouldn't raise this question again.